sci-calculators/qalculate-qt/qalculate-qt-5.2.0-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Bump with sci-libs/libqalculate and sci-calculators/qalculate-gtk!
+
+inherit qmake-utils xdg
+
+DESCRIPTION="Qt-based UI for libqalculate"
+HOMEPAGE="https://github.com/Qalculate/qalculate-qt"
+SRC_URI="https://github.com/Qalculate/${PN}/releases/download/v${PV}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+DEPEND="
+	dev-qt/qtbase:6[gui,network,widgets]
+	>=sci-libs/libqalculate-${PV}
```

Comment:
libqalculate has subslots:
```
# eshowkw libqalculate
Keywords for sci-libs/libqalculate:
         |                             |   u        |  
         | a   a     p s   a l   r     |   n        |  
         | m   r h   p p   l o m i s m | e u s      | r
         | d a m p p c a x p o i s 3 6 | a s l      | e
         | 6 r 6 p p 6 r 8 h n p c 9 8 | p e o      | p
         | 4 m 4 a c 4 c 6 a g s v 0 k | i d t      | o
---------+-----------------------------+------------+-------
   4.5.0 | + ~ + ~ ~ + + + ~ ~ o ~ o o | 8 # 0/22   | gentoo
   4.6.1 | + ~ + ~ ~ + + + ~ ~ o ~ o o | 8 #        | gentoo
   4.7.0 | ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ o ~ o o | 8 #        | gentoo
   4.8.0 | ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ o ~ o o | 8 #        | gentoo
   4.8.1 | + ~ + ~ + + + + ~ ~ o ~ o o | 8 o        | gentoo
   5.0.0 | ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ o ~ o o | 8 o        | gentoo
---------+-----------------------------+------------+-------
  5.0.0b | ~ ~ ~ ~ ~ ~ ~ ~ ~ ~ o ~ o o | 8 o 0/23   | gentoo
---------+-----------------------------+------------+-------
[I]5.2.0 | + ~ + ~ + + + + ~ ~ o ~ o o | 8 o 0/23.2 | gentoo
```

So unless being pinned down to exactly depend on same version of libqalculate, this is missing a slot rebuild operator. Taking a look at `sci-calculators/qalculate-gtk`, it depends on `>=sci-libs/libqalculate-${PV}:=`, so this should too.

Reply:
Fixed!